redirects.yaml

```diff
@@ -42,6 +42,30 @@ redirects:
 - from: /docs/webstore/pricing/
   to: /docs/webstore/money/
 
+# CWS payments/apps purge
+- from: /docs/webstore/hosted_apps
+  to: /docs/extensions/webstore
```

Comment:
I don't think this is the page you meant to link to as it's not a valid URL. Attempting to visit this URL results in a redirect to https://developer.chrome.com/docs/extensions/reference/webstore/. Did you perhaps mean to link to `/docs/webstore/`?

Reply:
Indeed I did. Fixed.